### PR TITLE
writing a readme file

### PR DESCRIPTION
**Pull request by Khaledhn1**
I was bored so I made this ReadMe file with the basic info about the Engine

### CODE REVIEW

**Comment by Khaledhn1** [comment]
@sagitarious12 alright I also added the links as you suggested

**Comment by saberslay** [comment]
I think that this will make the read me file better to see all of the supported/not supported  platforms

##### 
##### 
##### 
##### 
#####

**Comment by Khaledhn1** [comment]
> 
> 
> I think that this will make the read me file better to see all of the supported/not supported platforms
> 
> ```
> ##### 
> ##### 
> ##### 
> ##### 
> ##### 
> ```

good idea i implemented it and now the readme uses these images

**Comment by Gaztin** [comment]
It's great to see this kind of enthusiasm, however I think it feels a little weird for collaborators to write the readme for the project. This part especially: https://github.com/TheCherno/Hazel/blob/f896aad1087a27149f988dd89babced06d34fc6c/README.md#L5

**Comment by Gaztin** [comment]
That said, I do agree that the project needs a proper README

File: README.md
Comment (Puddlestomper) [inline README.md]:
```suggestion
The Hazel Engine is a 3D Application platform made by Yan "TheCherno" Chernikov, an Australian YouTuber who creates programming tutorials around different subjects from game development to general language tutorials.
```

File: README.md
Comment (Puddlestomper) [inline README.md]:
```suggestion
Consider checking out [TheChernoProject](https://www.youtube.com/user/TheChernoProject) on YouTube.
```

File: README.md
Comment (Puddlestomper) [inline README.md]: This sentence requires approval from Cherno before being merged.
```suggestion
Also any support on [Patreon](https://www.patreon.com/thecherno) is always appreciated.
```

File: README.md
Comment (Puddlestomper) [inline README.md]:
```suggestion
Currently only Windows is supported with plans to support Linux, MacOS and maybe Android or iOS in the future.
```

File: README.md
Comment (Puddlestomper) [inline README.md]: Maybe also add something about premake supporting other IDEs too?

```suggestion
##### 3. Open the project in Visual Studio
```

**Review by Puddlestomper — CHANGES_REQUESTED** [review]
Just some language things, and then the fact that Cherno's approval is required before this can be merged.

**Comment by LovelySanta** [comment]
I'll say to not merge this, but instead use the readme from #82. If you want so, feel free to copy paste that one in a new PR that closes this one (and the issue related to this one).

**Comment by Puddlestomper** [comment]
Seeing as Cherno made a readme file, this should either be closed or the features in this readme that are not in Cherno's one should be readded to this PR after merging in Cherno's one (which would mostly overwrite this readme I think?).

**Comment by LovelySanta** [comment]
Yes, this can indeed be closed as cherno did write a readme. We will update the one in 82, but that is part of that PR and not this one.

Closing as this is resolved

**Comment by hackerfinder64** [comment]
sabreslay is a hacker

**Pull request closed without merging**
